src/components/resultIcone.js

```diff
@@ -1,10 +1,3 @@
-import React from 'react';
-export class ResultIcone extends React.Component {
-    constructor(props){
-        super(props)
-        this.props = props;
-    }
-    render(){
-        return <div className = "calculator-result"> <span> {this.props.total}</span> </div>
-    }
+export const ResultIcone = (props) => {
```

Comment:
[OPTIONAL] When trying to use the modulo with 0, it throws an error. It should behave just the same it does the division by 0. You could try to solve this issue so the Calculator behaves as expected in every case.
![image](https://user-images.githubusercontent.com/79114233/143475748-12265056-3e4c-42e3-a5b5-90639f153b79.png)